### DISCUSSION TITLE
I have a piece of code, but it doesn't work as I expected. Could you tell me why?

### DISCUSSION THREAD

**Discussion by xiang-1208** [KaTeX/KaTeX#3392] [Q&A]
I have a piece of code, but it doesn't work as I expected. Can you tell me why? (Stupid though it is)

世界坐标系设定为 $\{W\}$，是一个在 $\{L\}$ 初始位置重合的 3D 坐标系。点 $i$，$i \in \cal{P}_k$ 在 $\{L\}$ 中表示为 $\boldsymbol{X}_{(k, i)}^{W}$。

It is shown as

![QQ截图20211102154220](https://user-images.githubusercontent.com/54844191/139805718-17038ca0-cce5-4f5d-8a13-8ca98305b53a.jpg)

**Comment by edemaine** [comment]
It looks like the two underscores (for subscripts) are getting interpreted by your Markdown parser as italics. You could escape them (use `\_`), or use a Markdown plugin that supports KaTeX directly.
